### ISSUE TITLE
do you trained this network?

### ISSUE THREAD

**Issue by Jmx5200** [pby5/MobileNet_Caffe#1]
do you trained this network，can the network converge?

**Comment by pby5** [comment]
yes, and I tuning parameters many times

**Comment by Jmx5200** [comment]
@pby5 if so，is there any access to the pre-trained models?

**Comment by pby5** [comment]
now I train with webface dataset but not with imagenet

**Comment by Ai-is-light** [comment]
why your BatchNorm did not set the " use_global_stats ：false" in your   train.prototxt  thanks?? @pby5 
layer {
	name: "conv1/bn"
	type: "BatchNorm"
	bottom: "conv1"
	top: "conv1/bn"
	param {
		lr_mult: 0
		decay_mult: 0
	}
	param {
		lr_mult: 0
		decay_mult: 0
	}
	param {
		lr_mult: 0
		decay_mult: 0
	}
}
